keyboards/hhkb_lite_2/config.h

```diff
@@ -0,0 +1,37 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1337
```

Comment:
This Vendor ID is already in use.  Please change:
https://www.the-sz.com/products/usbid/index.php?v=0x1337&p=&n=